src/dist/replication/lib/replica_learn.cpp

```diff
@@ -749,10 +750,11 @@ void replica::on_copy_remote_state_completed(
             else
             {
                 derror(
-                    "%s: on_copy_remote_state_completed[%016llx]: learner = %s, learn duration = %" PRIu64 " ms, apply learned state from private log failed, err = %s, duration = %" PRIu64 " ns",
-                    name(), req.signature, req.learner.to_string(),
-                    err.to_string(),
+                    "%s: on_copy_remote_state_completed[%" PRIx64 "]: learner = %s, "
```

Comment:
I was trying to change all to PRIx64 to be more portable, but in the end I changed only this place to avoid more conflicts for the ongoing merge we have and plan to do that later after merge. Anyway, this change is mostly about the wrong order of the parameters in the "else" branch that lead to crash.